seacatauth/credentials/handler.py

```diff
@@ -230,6 +230,18 @@ async def list_credentials(self, request):
 			L.error("Unsupported filter mode", struct_data={"mode": mode})
 			raise aiohttp.web.HTTPBadRequest()
 
+		#  Add brief tenant and role info into credentials list
+		for cred in credentials:
+			role_svc = self.CredentialsService.App.get_service("seacatauth.RoleService")
+			roles = await role_svc.get_roles_by_credentials(cred["_id"])
+			tenants = await self.TenantService.get_tenants(cred["_id"])
+
+			tenants = tenants[:5]
+			roles = roles[:5]
```

Comment:
You should not limit the return by 5, it could be very misleading for people using this call and not seeing tenants they are part of.
Honestly, this should be asked by frontend in a dedicated call(s) since this construct here can quickly become unsustainable.

Reply:
@ateska So this tenant/role aggregation should not be part of the `GET /credentials` response at all? 
Should it be up to frontend to make three separate calls and aggregate the results into one data table?

Reply:
Yes, it is a frontend task ... and eventually eg. GraphQL.